ipatests/test_integration/test_installation.py

```diff
@@ -2123,3 +2123,69 @@ def test_hostname_matching_domain(self):
         assert result.returncode == 1
         assert 'hostname cannot be the same as the domain name' \
             in result.stderr_text
+
+
+class TestNsslapdIgnoreTimeSkew(IntegrationTest):
+    """
+    Test to check nsslapd-ignore-time-skew is not disabled.
+    """
+    num_replicas = 1
+    topology = 'line'
+
+    @classmethod
```

Comment:
Does the test really need to override the method? IntegrationTest already takes care of server and replica installation, unless you want to specify different arguments.